model_analyzer/config/input/config_list_numeric.py

```diff
@@ -103,7 +103,14 @@ def set_value(self, value):
         try:
             if self._is_string(value):
                 self._value = []
-                value = value.split(",")
+                if "," in value:
+                    value = value.split(",")
+                elif ":" in value:
```

Comment:
Is there reason why we can't use comma as a separator for periodic concurrency? Seems like it was already using comma for other inference loads

Reply:
This is how it's documented in PA:
```--periodic-concurrency-range <start:end:step>```

Just wanted to keep us consistent.

Reply:
Is the `ConfigListNumeric` class still used for periodic concurrency? I see that periodic concurrency is using `ConfigListString`

Reply:
No, it's using `ConfigListString`. The change in `ConfigListNumeric` is for non-periodic concurrency. So, the user can specify <start:end:step> without needing to create a dictionary: `--concurrency 5:50:5`

For periodic concurrency they specify a string like: `"5:50:5", "10:100:10"` which can contain multiple sets of <start:stop:end> (vs. ConfigListNumeric which only allows a single set).